result_quality.md

```diff
@@ -0,0 +1,56 @@
+# Determining Result Quality
+
+Pelias tries very hard to always return good results, and generally does a pretty good job. If using the autocomplete endpoint or otherwise showing results directly to humans, judging result quality is generally easy (if subjective).
+
+However, when programmatically managing results from Pelias, determining result quality is much harder. Pelias returns several different properties with each result to help determine if a result is good enough for your purposes.
```

Comment:
each result *contains* several different properties?

how would you feel about adding 'air quotes' around things like 'good enough'?